index.html

```diff
@@ -140,6 +140,10 @@ <h2>Page Layout</h2>
                 </span> 
                 <span class="row"> <label>Add Cutlines <input type="checkbox" name="cutmarks"></label>
                     <span data-balloon-length="medium" aria-label="Adds cutting guidelines" data-balloon-pos="up">ℹ️</span>
+                
+                </span>
+                <span class="row"> <label>Add PDF bounds indicators (spine) <input type="checkbox" name="pdf_edge_marks"></label>
+                    <span data-balloon-length="medium" aria-label="Puts small marks on the outermost folio of the signatures indicating the top and bottom of the PDF" data-balloon-pos="up">ℹ️</span>
```

Comment:
This is intended to be for trimming, right? Might be worth making that clearer in the description.

Reply:
Doesn't have to be? I don't want to add too much details (am coming back in shortly to add spine lines) 
I actually just found it very useful for finding the outermost folio and being certain they were lined up correctly (since I put uneven margins top &  bottom ID them... )
Will add "Useful for trimming" at least to popup